ch_tools/common/clickhouse/config/users.py

```diff
@@ -18,4 +18,6 @@ def dump_xml(self, mask_secrets=True):
 
     @staticmethod
     def load():
-        return ClickhouseUsersConfig(load_config(CLICKHOUSE_USERS_CONFIG_PATH))
+        return ClickhouseUsersConfig(
+            load_config(CLICKHOUSE_USERS_CONFIG_PATH, "users.d")
+        )
```

Comment:
This is the main change. The rest is a test and supporting changes for the test.